README.md

```diff
@@ -125,7 +131,7 @@ The options are:
   The AWS secret of the provided key.
 
 ##### `region` ("us-east-1")
-  The AWS region the S3 bucket is located in.
+  The AWS region the S3 bucket is located in. If you are getting `The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint.` error, specify your bucket `--region`.
```

Comment:
Please add two newlines between the existing text and your change.  So it becomes:

```
... located in.

If you are getting...
```